pkg/userdata/ubuntu/provider.go

```diff
@@ -247,6 +247,7 @@ write_files:
     swapoff -a
 
     export CR_PKG='docker-ce=5:18.09.9~3-0~ubuntu-bionic'
```

Comment:
For k8s 1.17 should we also pin containerd to version `1.2.10-3` as described [here?](https://v1-17.docs.kubernetes.io/docs/setup/production-environment/container-runtimes/#docker)

Reply:
I think that exporting and env variable here is useless, why don't we put the package directly in the install command?

Reply:
> I think that exporting and env variable here is useless, why don't we put the package directly in the install command?

I was just following what has been done for `CR_PKG`. It's the same situation and I'm not really sure why we have that environment variable.

> For k8s 1.17 should we also pin containerd to version 1.2.10-3 as described here?

I agree that we should bind containerd as well, but we need to find the compatibility matrix, which I wasn't able to find.

Reply:
> I was just following what has been done for CR_PKG. It's the same situation and I'm not really sure why we have that environment variable.

Not sure too, but there's nothing relying on it in the script, I think we can get rid of it safely. If you don't want to do it in this PR I will open a follow-up PR.

> I agree that we should bind containerd as well, but we need to find the compatibility matrix, which I wasn't able to find.

I see your point, I did not find supported versions in k8s release notes too. Let's keep it unpinned for the moment.